src/ArduinoIoTCloud.cpp

```diff
@@ -418,6 +412,7 @@ void ArduinoIoTCloudClass::connectionCheck() {
 
 void ArduinoIoTCloudClass::setIoTConnectionState(ArduinoIoTConnectionStatus _newState) {
```

Comment:
just a comment on style:
if we use an underscore prefix to characterise member variables we should not use them in signatures.
I have been always used to characterise with underscore my signature ones, and it's very possible I wrote this method in the first place.
let's pick one, but since I've seen underscores in all the member properties we should change this one